modules/conversion-center/classes/render/render-base.php

```diff
@@ -233,14 +233,24 @@ protected function render_icons(): void {
 					'class' => $icon_class_names,
 				] );
 
-				$this->widget->add_render_attribute( 'icon-link-' . $key, [
+				// Manage Link attributes
+
+				$url_attrs = [
 					'aria-label' => esc_attr( $icon['icon_platform'] ),
 					'class' => 'e-link-in-bio__icon-link',
 					'href' => esc_url( $formatted_link ),
-					'rel' => 'noopener noreferrer',
-					'target' => '_blank',
-				] );
+				];
 
+				$url_combined_attrs = $this->get_link_attributes(
+					$icon['icon_url'],
+					$url_attrs
+				);
+
+				foreach ( $url_combined_attrs as $attr_key => $attr_value ) {
+					$this->widget->add_render_attribute( 'icon-link-' . $key, [
+						$attr_key => $attr_value,
```

Comment:
```suggestion
						$attr_key => esc_attr( $attr_value ),
```
Can you check if this is needed?

Reply:
@louiswol94 

These are output by get_render_attribute_string()

Which in turns uses render_html_attributes()
As far as I can see attributes are escaped there so this should be fine
https://github.com/elementor/elementor/blob/11b7ca53fe87940971b8aeb103a250f292b1e19e/includes/utils.php#L501C1-L513C3